examples/plot_spm_auditory.py

```diff
@@ -6,6 +6,7 @@
 """
 
 import os
+import os.path as op
```

Comment:
I am not going to laugh at anyone but since you are using it once below, you may as well remove this import and use the more verbose os.path.